docs/UserGuide.md

```diff
@@ -43,144 +42,260 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 
 ### 2.1. Viewing help : `help`
 
-Format: `help`
+Format: `help [COMMAND]`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+> * When there is no command specified, `help` will print help messages of all commands.
+> * Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task: `add`
```

Comment:
The "n/" may really be removable! We will consider about that!
We would actually want to give more convenience to our user in a sense that they can choose to enter date in many different formats, for example "today", "tomorrow", "next Wednesday", "13/11/17", "8 Mar".
The field "*f" may not has been explained. Will fill up the gap!